tests/apollo/util/skvbc.py

```diff
@@ -330,6 +331,20 @@ async def read_your_writes(self, test_class):
 
         print(f'[READ-YOUR-WRITES] OK.')
 
+    async def wait_for_system_to_be_live(self, time_to_wait=60):
+        """Wait for system to be able to reach quorums and execute clients' requests"""
+        with trio.fail_after(seconds=time_to_wait):
+            while True:
+                with trio.move_on_after(seconds=5):
+                    try:
+                        k, v = await self.write_known_kv()
```

Comment:
Unfortunately, I believe this approach might not be very reliable... According to my findings here - https://jira.eng.vmware.com/browse/BC-3028, sending requests while the active window is not processed by consensus yet, might trigger additional view changes!